Document Azure ServiceBus access right requirements (#6017)
Particular/docs.particular.net: transports/azure-service-bus/configuration_access-rights_asbs_[1.0,3.1].partial.md

### Access rights

> By default the transport requires elevated privileges to manage subscriptions at runtime. If using a [shared access policy](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-sas) make sure to include `Manage` rights and the [Azure Service Bus Data Owner](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#azure-service-bus-data-owner) role if authenticating using [Managed Identities](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-managed-service-identity).

Reviewer: ```suggestion
By default the transport requires elevated privileges to manage subscriptions at runtime. If using a [shared access policy](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-sas) make sure to include `Manage` rights or the [Azure Service Bus Data Owner](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#azure-service-bus-data-owner) role if authenticating using [Managed Identities](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-managed-service-identity).
```

To avoid running with managed rights:

> - Make sure that [installers are not configured to run](/nservicebus/operations/installers.md)
> - [Turn off automatic subscriptions](/nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md#disabling-auto-subscription)
> - Use [operational scripting](/transports/azure-service-bus/operational-scripting.md) to provision entities(queues, topics and subscriptions)

Reviewer: ```suggestion
- Disable [installers](/nservicebus/operations/installers.md)
- [Turn off automatic subscriptions](/nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md#disabling-auto-subscription)
- Use [operational scripting](/transports/azure-service-bus/operational-scripting.md) to provision queues, topics and subscriptions
```

Author: Aren't installers off by default?

Author: They are off: https://docs.particular.net/nservicebus/operations/installers#running-installers
